particles/PipeApps2/PeopleAutofill.recipes

```diff
@@ -9,8 +9,8 @@ particle SuggestPerson in './source/SuggestPerson.js'
 particle RequireQuery in './source/Noop.js'
   in PipeEntity query
 
-recipe PeopleAutofill &autofill
-  use #autofill_com_google_android_gm as query
+recipe PeopleAutofill &artist_autofill
+  use #artist_autofill as query
```

Comment:
PeopleAutofill !== artist_autofill, something non-obvious is happening here.

Reply:
reverted this file (i was hacking with the recipes for a demo, and this was a leftover of my confusion, sorry)